### ISSUE TITLE
VRF support for eos_eapi

### ISSUE THREAD

**Issue by advornic** [arista-eosplus/ansible-eos#12]

**Comment by xigxag11** [comment]
Is there currently no way to use eos_eapi with Ansible when the api is running in a non-default VRF? I am able to bind the eAPI https server to the VRF and connect via browser, but my playbook runs will then fail on eAPI.

Is there any known work-around?


```
TASK: [run an arbitrary EOS command] ****************************************** 
failed: [10.29.230.100] => {"failed": true, "parsed": false}
SUDO-SUCCESS-cwfjavcqyyokhcrceaesehojnrkcxouf
Traceback (most recent call last):
  File "/persist/local/ansible/.ansible/tmp/ansible-tmp-1422647224.05-28406876568750/eos_command", line 1712, in <module>
    main()
  File "/persist/local/ansible/.ansible/tmp/ansible-tmp-1422647224.05-28406876568750/eos_command", line 1703, in main
    resp = func(commands)
  File "/usr/lib/python2.7/site-packages/eapilib/connections.py", line 146, in enable
    raise ConnectionError(e)
eapilib.connections.ConnectionError: [Errno 111] Connection refused
```

**Comment by privateip** [comment]
This issue will be fixed in the coming 1.0 release.

**Comment by phil-arista** [comment]
Release Note:
Tested this with the latest code and not an issue.  The eos_eapi module has been deprecated so the only issue was to confirm that pyeapi can connect to eapi when eapi in configured in a VRF and the mgmt interface is also in that VRF.  This as tested and works.
